tests/app/celery/test_broadcast_message_tasks.py

```diff
@@ -59,3 +60,59 @@ def test_send_broadcast_message_errors(sample_service):
             send_broadcast_message(broadcast_message_id=str(bm.id))
 
     assert ex.value.response.status_code == 503
+
+
+@freeze_time('2020-08-01 12:00')
+def test_send_broadcast_event_sends_data_correctly(sample_service):
+    t = create_template(sample_service, BROADCAST_TYPE)
+    bm = create_broadcast_message(t, areas=['london'], status=BroadcastStatusType.BROADCASTING)
+    be = create_broadcast_event(bm)
```

Comment:
do you think it would be ok to have more readable variable names, like words, to reduce things to keep in memory when reading the tests?